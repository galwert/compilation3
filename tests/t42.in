
int func() {
    return 1;
}

int ags() {
    return 1;
}

int zxvc() {
    return 1;
}

int sdg() {
    return 1;
}

int qwr(bool x1, bool x2) {
    return 1;
}
void main(){
    printi(func); // Yes, I am reusing tests. Don't judge
}


bool sdfa4(bool x1, bool x2) {
    return x1 or x2;
}

voir nbfdrct(bool x1, bool x2) {
    int x = 1;
    byte y = 2 b;

    return;
}

int cwrvsgdhs(bool x1, bool x2, byte x0) {
    return 1;
    int x3;
    int x4;
    int x5;
}

int ytyjbd(bool x1, bool x2) {
    return 1;
}

int afsdtv(bool x1, bool x2) {
    return 1;
}

