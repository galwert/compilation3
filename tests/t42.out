---end scope---
---end scope---
---end scope---
---end scope---
---end scope---
x1 BOOL -1
x2 BOOL -2
line 22: variable func is not defined
